modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/ActionCreationForm.tsx

```diff
@@ -49,20 +50,25 @@ const TYPES = [
 ];
 
 interface IFDSActionCreationFormInterface {
+	activeSection: string;
```

Comment:
It's a bit mind-bending to me to have such specific component reused in two sections. Typical hierarchy would be a single section (ITEM_ACTION), a single `ActionCreationForm`, with elements modified by some add-or-edit flag in prop (`initialValues`). But, this might be just me :shrug:

Reply:
Good point! I'll add a prop called `editing` instead.

Reply:
Re-reading this, I misunderstood fully at first...I'll think about the possibility of a single section.

Reply:
I've changed `activeSection` to an `editing` prop so there's no dependency on the active section being set. I still like the re-use for 2 sections to keep the "new" form clean with no accidental initial values.